README.md

```diff
@@ -11,6 +11,135 @@ supported boards and instructions on how to setup environment and build images.
 
 ![Mender logo](https://mender.io/user/pages/05.resources/06.digital-assets/logo.png)
 
+## Structure
+
+This section is for anyone interested in contribution support for an board.
+
+meta-mender-community is a top level layer which is not actually a layer it
+self but contains a collection of layers which integrates Mender on various
+boards.
+
+The layers are structured based on upstream BSP layers and not individual
+boards.
+
+There are several types of BSP layers, some are SoC oriented and a selection of
+those would be:
+
+- meta-freescale
+- meta-sunxi
+- meta-rockchip
+- meta-renesas
+
+And then you also have System on Module vendors that typically have their own
+BSP layers which append to the SoC ones, and a selection of those would be:
+
+- meta-variscite
+- meta-phytec
+- meta-toradex-nxp
+
+Naming of integration layers in meta-mender-community follows the upstream
+naming closely where SoM vendor layer name take precedence and the convention
+is:
+
+   meta-mender-<upstream suffix>
+
+This should make it clear which layer is targeting what BSP.
+
+### Manifest files
+
+We utilize the [google repo](https://source.android.com/setup/develop/repo) tool
```

Comment:
The google repo tool and associated manifest files are used for managing the list of repositories needed for these builds. The common manifest (meta-mender-community/scripts/mender.xml) is included by board-specific manifests and include the required Mender layers: